go/vt/vttablet/onlineddl/executor.go

```diff
@@ -890,7 +890,18 @@ func (e *Executor) ExecuteWithVReplication(ctx context.Context, onlineDDL *schem
 	if err := e.updateMigrationTableRows(ctx, onlineDDL.UUID, v.tableRows); err != nil {
 		return err
 	}
-	if err := e.updateMigrationAddedRemovedUniqueKeys(ctx, onlineDDL.UUID, len(v.addedUniqueKeys), len(v.removedUniqueKeys)); err != nil {
+	removedUniqueKeyNames := []string{}
+	for _, uniqueKey := range v.removedUniqueKeys {
+		removedUniqueKeyNames = append(removedUniqueKeyNames, uniqueKey.Name)
+	}
+	if err := e.updateSchemaAnalysis(ctx, onlineDDL.UUID,
+		len(v.addedUniqueKeys),
+		len(v.removedUniqueKeys),
+		strings.Join(removedUniqueKeyNames, ","),
```

Comment:
Is it safe to concatenate like this? MySQL lets you use commas in column names.

Reply:
You are right -- but this is going to be informational only. That is, we're not going to then re-parse this. This is about us being able to tell the user "there is a chance we can't revert this migration due to these columns: a,b,c". So in a way, I consider this to be free text.
However, you're right and why no make the extra effort of qualifying the names. I'll proceed to do that.